src/google-chart/google-chart.component.ts

```diff
@@ -44,35 +50,139 @@ export class GoogleChartComponent implements OnChanges {
+  * a column of type number means a series column.
+  * @param column to inspect
+  */
+  private getSeriesByColumn(column:number): number  {
+    let series: number = 0;
+    let dataTable = this.wrapper.getDataTable();
+    for(let i=column-1; i>=0; i--) {
+      let role = dataTable.getColumnRole(i);
+      let type = dataTable.getColumnType(i);
+      if(role === 'data' || type === 'number' ) {
+        series++;
+      }
+    }
+    return series;
+  }
+
+  private getBoundingBoxForItem(item: DataPointPosition): BoundingBox {
+    let boundingBox = {top : 0, left:0, width:0, height:0};
+    if(this.cli) {
+      let column = item.column;
+      let series = this.getSeriesByColumn(column);
+      let bar = item.row;
+      let row = item.row;
+
+      if(this.options.series && this.options.series[series] && (this.options.series[series].type || this.options.seriesType) ) {
+        let seriesType = this.options.series[series].type || this.options.seriesType;
+        let selector = this.getSelectorBySeriesType(seriesType);
+        if(selector) {
+             selector = selector.replace('%s',series + '').replace('%c',column+'').replace('%r',row+'');
+             let box = this.cli.getBoundingBox(selector);
+             if(box) {
+              boundingBox = box;
+             }
+        }
+      }
+    }
+
+    return boundingBox;
+  }
+
+  private getDataValueAtPosition(position: DataPointPosition):any {
+    return {};
+  }
+
+  private getDataTypeAtPosition(position: DataPointPosition):string {
+    return '';
+  }
+
+  private getHTMLTooltip(): ChartHTMLTooltip {
+    let tooltipER = new ElementRef(this.el.nativeElement.querySelector('.google-visualization-tooltip'));
+    return new ChartHTMLTooltip(tooltipER);
+  }
+
+  private parseDataPointHoveredEvent(item: DataPointPosition): DataPointHoveredEvent {
+        let event = {
+          hoveredItemPosition: item,
+          hoveredItemBoundingBox: this.getBoundingBoxForItem(item),
+          hoveredItemValue: this.getDataValueAtPosition(item),
+          tooltip: this.getHTMLTooltip(),
+          hoveredItemType: this.getDataTypeAtPosition(item)
+        };
+        return event;
+  }
+
+  private unregisterChartEvents():void {
+    let chart = this.wrapper.getChart();
+    google.visualization.events.removeAllListeners(chart);
+  }
+
   private registerChartEvents(): void {
+    if(this.onMouseOver.observers.length > 0 ) {
+      let chart = this.wrapper.getChart();
+      this.cli = chart.getChartLayoutInterface();
+      google.visualization.events.addListener(chart, 'onmouseover', (item: DataPointPosition) => {
+        let event = this.parseDataPointHoveredEvent(item);
+        this.onMouseOver.emit(event);
+      });
+    }
+  }
+
+  private registerChartWrapperEvents(): void {
+
     google.visualization.events.addListener(this.wrapper, 'ready', () => {
       this.chartReady.emit({message: 'Chart ready'});
+      this.registerChartEvents();
```

Comment:
Chart events can only be registered after the chart is drawn and ready. Here we wait for the ready event to happen before register the chart events (not the wrapper events)